.circleci/triggers.py

```diff
@@ -13,15 +13,15 @@
 [gpu]: Run the nightly GPU tests
 [mturk]: Run the mturk tests
 [data]: Run the data tests
-[long]: run all above
+[long] or [all]: run all above
 """
 
 import parlai.core.testing_utils as testing_utils
 
 
 def detect_all():
     """Check if we should run all tests."""
-    return '[long]' in testing_utils.git_commit_messages()
+    return any(kw in testing_utils.git_commit_messages() for kw in ['[all]', '[long'])
```

Comment:
nit: believe you want `'[long]']` rather than `'[long']` (though i suppose it'll work regardless)

Reply:
lol good catch